icu4j/main/classes/translit/src/com/ibm/icu/text/AnyTransliterator.java

```diff
@@ -52,6 +52,12 @@ class AnyTransliterator extends Transliterator {
     static final String NULL_ID = "Null";
     static final String LATIN_PIVOT = "-Latin;Latin-";
 
+    /**
+     * Special code for handling width characters
+     */
+    private static final Transliterator WIDTH_FIX =
```

Comment:
optional: Is this even commonly used? If not, then it might be worth moving this constant into a nested class, in order to defer initialization until it is actually used.

If it is commonly used, then that's not worth it.

Reply:
Given that it's always initialized in <clinit> even before this PR, Is it okay to submit this PR first?  I will take a look at this along with other ICU4C issues mentioned in this PR.

Reply:
If it's okay, could you help merge this PR? As I am not authorized to merge it

Reply:
https://github.com/unicode-org/icu/pull/2491 for moving it into the nested class.